relayer/src/link/operational_data.rs

```diff
@@ -78,18 +78,25 @@ pub struct TransitMessage {
     pub msg: Any,
 }
 
-/// Holds all the necessary information for handling a set of in-transit messages.
+/// Holds all the necessary information for handling a batch of in-transit messages. This includes
+/// an event received from a chain along with any other packets related to the event (i.e.
+/// 'receive' or 'timeout' packets) that the relayer has to submit in response to the event.
 ///
-/// Each `OperationalData` item is uniquely identified by the combination of two attributes:
-///     - `target`: represents the target of the packet messages, either source or destination chain,
-///     - `proofs_height`: represents the height for the proofs in all the messages.
-///       Note: this is the height at which the proofs are queried. A client consensus state at
-///       `proofs_height + 1` must exist on-chain in order to verify the proofs.
+/// Each `OperationalData` item is uniquely identified by the combination of its `target` chain
+/// and its `proofs_height` value.
```

Comment:
With the introduction of the `tracking_id` field, does this still hold true? I would think that the `tracking_id` is now the main source of unique identification for a piece of operational data.

Reply:
Context: The `tracking_id` adds the ability to track a set of events from the moment Hermes receives them via websocket until it submits the transactions corresponding to those events. An operational data inherits the `tracking_id` from the events it contains. Each `tracking_id` should be unique, but has no semantic meaning.

> With the introduction of the tracking_id field, does this still hold true?

It's still true, but now I realize it's not a very useful piece of information, it may even be confusing, so cool to remove it!